KukaApp/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { StyleService, Text, useStyleSheet } from '@ui-kitten/components';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+const themedStyles = StyleService.create({
+  headerContainer: {
+    backgroundColor: 'color-primary-default',
+    fontFamily: 'OpenSans-Bold',
+    fontWeight: '700',
+    fontSize: 26,
+    paddingLeft: 18,
+    paddingBottom: 10,
+  },
+});
+
+export const Header = ({ text }) => {
+  const insets = useSafeAreaInsets();
+  const styles = useStyleSheet(themedStyles);
+
+  return (
+    <Text
+      style={{ ...styles.headerContainer, paddingTop: insets.top }}
+      category="h2"
+      appearance="alternative"
+    >
+      {text}
```

Comment:
Thoughts on just using children to behave similar to UI Kitten's and React Native's `<Text>Meow</Text>` instead of `<Text text="Bark"></Text>`

so doing something like

```
<Text
      style={{ ...styles.headerContainer, paddingTop: insets.top }}
      category="h2"
      appearance="alternative"
    >
      {props.children}
</Text>
```

Reply:
There's actually a part of me that wants to pull out the stylesheet into its own file.